types/minio/index.d.ts

```diff
@@ -247,6 +247,11 @@ export interface MakeBucketOpt {
     ObjectLocking: boolean;
 }
 
+export interface RemoveOptions {
```

Comment:
nit: I think this would help with ambiguity of other available `remove` fns

```suggestion
export interface RemoveObjectOptions {
```